docs/pages/guides/webhooks.mdx

```diff
@@ -0,0 +1,196 @@
+
+In this way, webhooks provide a real-time connection between two services. While
+APIs are useful for obtaining data on a scheduled basis, they are not efficient
+for observing changes on a platform that has a high volume of activity.
+
+### Concepts
+
+## Messages
+
+For each activity that occurs under the defined list of events within the Event
+Catalog a message is generated. Messages contain the metadata for each event
+that you may choose to subscribe to. They also contain event data and an
+alphanumeric identifier.
+
+## Events
+
+Messages sent through Svix each have an `eventType` property which is used to
+identify and filter what is being sent to the client. Each endpoint you provide
+in the webhooks dashboard listens to all events by default but can be easily
+configured to only listen to a subset by updating the Message Filtering section.
+As a new user, you may opt to listen to all events or subscribe to only the ones
+that are relevant to your use case.
+
+Alternatively, you can use the Event Catalog provided as a way to determine
+which events make the most sense. The Event Catalog provides a list of events
+available for subscription, along with their schema.
+
+The example schema for monitoring users joining a room:
+
+```typescript
+type UserEntered {
+	type: "userEntered"
+	data: {
+		roomId: string
+		appId: string
+		connectionId: number // actor
+		userId: string | null
+		userInfo: Record<string, any> | null
+		joinedAt: string // ISO 8601 UTC // 2021-10-06T01:45:56.558+00:00z
+		numActiveUsers: number
+	}
+}
+```
+
+The sample payload of such an event:
+
+```typescript
+const UserEnteredMessage = {
+	type: "userEntered",
+	data: {
+		appId: "aasdbe234bsdf"
+		roomId: "my-room-1"
+		numActiveUsers: 8
+		connectionId: 4
+		userId: '1234sdgs'
+		userInfo: null
+		enteredAt: '2021-10-06T01:45:56.558+00:00'
+	}
+}
+```
+
+## Endpoints
+
+An endpoint is a URL whose purpose is to receive event messages and trigger a
+task based on the message payload. The only criterion for creating an endpoint
+is that it must be one that you control.
+
+## Use Case
+
+Prior to webhooks, if you wanted to update internal data based on Liveblocks'
+state, you would need to set up a cron job to poll the
+[rooms API](https://development.liveblocks.io/docs/api-reference/rest-api-endpoints)
+for recurrent updates. With webhooks, you can now subscribe to the events you
+are interested in, and be alerted of the change when it happens. Powerful ways
+to leverage webhooks with Liveblocks include:
+
+- Storage Synchronization between room(s) and internal database
+- Alerting the client when a user leaves or enters a room
+- Notifying the client if max concurrency has been reached
+
+### Setup
+
+## Creating an endpoint
+
+Creating endpoints is simple, and is done from within the webhooks dashboard for
+your project.
+
+Before sending events, you should disable CSRF protection for the endpoint if it
```

Comment:
Is it evident to people where to go to disable CSRF? Could we give some directions or add a screenshot?

Reply:
ok saw you wanted to add a video below. SO I guess it will cover that?

Reply:
No it is something that is going to vary by implementation.  Disabling CSRF would probably be something that we would mention in a tutorial, as opposed to this guide since it would be framework specific.